cmd/mimir/config-descriptor.json

```diff
@@ -2116,20 +2116,42 @@
               "kind": "field",
               "name": "failure_threshold",
               "required": false,
-              "desc": "Max number of requests that can fail in a row before the circuit breaker opens",
+              "desc": "Max percentage of requests that can fail over period before the circuit breaker opens",
               "fieldValue": null,
               "fieldDefaultValue": 10,
               "fieldFlag": "ingester.client.circuit-breaker.failure-threshold",
               "fieldType": "int",
```

Comment:
If this field is a percentage now, should it be a float? (Would we want to be able to set the percentage at a more granular level than 1%?)

Reply:
It's an int, from 0-100 (the precedent for doing this with circuitbreakers came from [Hystrix](https://github.com/Netflix/Hystrix/wiki/Configuration#circuitBreaker.errorThresholdPercentage), FWIW). I'm sure 1% increments are granular enough. A harder problem is choosing a good value, but even a default like 10% is probably better than guessing at rate limits or concurrency limits.

Reply:
We could make it a float but the underlying library (failsafe) uses a `uint` from 0 to 100 so we'd have the same granularity at the end of the day.